pkgs/top-level/pidgin-packages.nix

```diff
@@ -0,0 +1,70 @@
+{ lib
+, newScope
+, pidgin
+, texlive
+, config
+}:
+
+lib.makeScope newScope (self:
+  let callPackage = self.callPackage;
+  in {
+    pidgin = callPackage ../applications/networking/instant-messengers/pidgin {
+      withOpenssl = config.pidgin.openssl or true;
+      withGnutls = config.pidgin.gnutls or false;
+      plugins = [];
+    };
+
+    pidginPackages = self;
+
+    pidgin-indicator = callPackage ../applications/networking/instant-messengers/pidgin-plugins/pidgin-indicator { };
```

Comment:
It should be inside `pidgin` directory too, in a simila way we do with Emacs Lisp packages.

Reply:
The top level emacs packages are in top-level

Only stuff about the different sources like ELPA and MELPA are in emacs folder

Reply:
> The top level emacs packages are in top-level

I am not talking about the logical place of the expressions, but physical location of files.

> 
> Only stuff about the different sources like ELPA and MELPA are in emacs folder

Yes, and it is better to put those files inside `pidgin/plugins` instead of `pidgin-plugins`, the same form the Elisp packages are inside `emacs/elisp-packages` instead of  `emacs-elisp-packages`

Reply:
But whatever, this can be changed later